charts/external-dns/Chart.yaml

```diff
@@ -20,4 +20,4 @@ maintainers:
 annotations:
   artifacthub.io/changes: |
     - kind: changed
-      description: "Update ExternalDNS version to v0.11.0"
+      description: "Update ExternalDNS version to v0.12.0"
```

Comment:
Could you add the other items in the CHANGELOG to the annotations?

Reply:
Done.

Reply:
I usually remove the markdown from these as ArtifactHub doesn't support markdown here.

Reply:
Ok, I will remove the markdown from there.

Reply:
Done.